src/styles/blocks/card.scss

```diff
@@ -0,0 +1,143 @@
+/* #region card */
+.card {
+  display: flex;
+  flex-direction: column;
+
+  margin: 0;
+  width: $card-width;
+
+  box-sizing: border-box;
+  border: $border-width solid rgba(243, 243, 243, 1);
+  border-radius: 5px;
+
+  &__img {
+    width: 160px;
+    height: 134px;
+
+    margin-left: 19px;
+    margin-top: 32px;
```

Comment:
Will be better to add correct padding to the card instead of such indents for image